kolibri/plugins/learn/assets/src/views/EmbeddedSidePanel/index.vue

```diff
@@ -307,9 +296,6 @@
     },
     methods: {
       genContentLink,
-      allCategories() {
-        this.$emit('input', { ...this.value, categories: { [AllCategories]: true } });
```

Comment:
This event will still need to be emitted in order for other parts of the app to continue working correctly. Instead of being emitted when the button is clicked, instead it should be emitted when the number of selected categories goes to zero.